gateway-service-impl/src/main/java/org/hypertrace/gateway/service/baseline/BaselineServiceImpl.java

```diff
@@ -80,13 +82,19 @@ public BaselineEntitiesResponse getBaselineForEntities(
       Period aggTimePeriod =
           getPeriod(originalRequest.getStartTimeMillis(), originalRequest.getEndTimeMillis());
       long periodSecs = getPeriodInSecs(aggTimePeriod);
-      List<TimeAggregation> timeAggregations = getTimeAggregationsForAggregateExpr(originalRequest);
+      long alignedStartTime =
```

Comment:
I think we should actually log a tech debt and check why do we even need this alignment in the first place. Might be doing unnecessarily too.

Reply:
https://traceableai.atlassian.net/browse/ENG-7586 logged a jira for that.

Reply:
yes, will log a jira.